javascript/javascript2/week1/exercise01.js

```diff
@@ -0,0 +1,92 @@
+//Write a function that finds the shortest word of an array of words
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const lengthOfArray = danishWords.length;
+function shortestWords(danishWords) {
+  for (let i = 0; i < lengthOfArray; i++) {
+    let arraysIndexLen = danishWords[i].length;
+    console.log(arraysIndexLen);
+    if (arraysIndexLen <= 1) {
+      return danishWords[i];
+    }
+  }
+}
+console.log(shortestWords(danishWords));
+
+//Find the individual number and the total number of Danish letters in a string.
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+let count = 0;
+let countå = 0;
+let countø = 0;
+let countæ = 0;
+function individualChar(danishString) {
+  for (let i = 0; i < danishString.length; i++) {
+    let stringsIndex = danishString[i];
+    console.log(stringsIndex);
```

Comment:
Try to use switch case in this type of conditions.

Reply:
Also Please read about Regx, it will help you in future to work on strings.

Reply:
I prefer this instead of switch case and I read about Regx but Frederik had advised not to use it much